doc/mgmt/SONiC_OC_PortChannel_Interface_HLD.md

```diff
@@ -0,0 +1,439 @@
+          |  |  +--ro oc-eth:hw-mac-address?           oc-yang:mac-address
+          |  |  +--ro oc-eth:negotiated-duplex-mode?   enumeration
+          |  |  +--ro oc-eth:negotiated-port-speed?    identityref
+          |  |  +--ro oc-eth:counters
+          |  |  |  +--ro oc-eth:in-mac-control-frames?    oc-yang:counter64
+          |  |  |  +--ro oc-eth:in-mac-pause-frames?      oc-yang:counter64
+          |  |  |  +--ro oc-eth:in-oversize-frames?       oc-yang:counter64
+          |  |  |  +--ro oc-eth:in-undersize-frames?      oc-yang:counter64
+          |  |  |  +--ro oc-eth:in-jabber-frames?         oc-yang:counter64
+          |  |  |  +--ro oc-eth:in-fragment-frames?       oc-yang:counter64
+          |  |  |  +--ro oc-eth:in-8021q-frames?          oc-yang:counter64
+          |  |  |  +--ro oc-eth:in-crc-errors?            oc-yang:counter64
+          |  |  |  +--ro oc-eth:in-block-errors?          oc-yang:counter64
+          |  |  |  +--ro oc-eth:out-mac-control-frames?   oc-yang:counter64
+          |  |  |  +--ro oc-eth:out-mac-pause-frames?     oc-yang:counter64
+          |  |  |  +--ro oc-eth:out-8021q-frames?         oc-yang:counter64
++         |  |  +--ro oc-lag:aggregate-id?             -> /oc-if:interfaces/interface/name
++         +--rw oc-lag:aggregation
++         |  +--rw oc-lag:config
+          |  |  +--rw oc-lag:lag-type?    aggregation-type
++         |  |  +--rw oc-lag:min-links?   uint16
++         |  +--ro oc-lag:state
+          |  |  +--ro oc-lag:lag-type?    aggregation-type
++         |  |  +--ro oc-lag:min-links?   uint16
+          |  |  +--ro oc-lag:lag-speed?   uint32
++         |  |  +--ro oc-lag:member*      oc-if:base-interface-ref
++         |  |  +--ro if-agmnt:fallback?  boolean  //Augmented
+
+```
+
+**Note:** openconfig-if-aggregate.yang data model is augmented to support LACP Fallback.
+
+### 3.6.2 CLI
+#### 3.6.2.1 Configuration Commands
+
+### Create a PortChannel
+`interface PortChannel <channel-number>`
+```
+sonic(config)# interface PortChannel 1
+```
+### Configure min-links
+`minimum-links <number>`
+<br>
+ Default:0
+```
+sonic(config)# interface PortChannel 1
+sonic(conf-if-po1)# minimum-links 1
+```
+### Reset to default value of min-links
+`no minimum-links`
+<br>
+ Default:0
+```
+sonic(conf-if-po1)# no minimum-links 
+```
+### Configure MTU
+`mtu <mtu-val>`
+```
+sonic(conf-if-po1)#  mtu 9000
+sonic(conf-if-po1)# no mtu
+```
+#### MTU Removal
+`no mtu` --> Reset to default
+```
+sonic(conf-if-po1)# no mtu
+```
+
+#### Enable PortChannel
+`no shutdown` 
+```
+sonic(conf-if-po1)# no shutdown
+```
+#### Disable PortChannel
+`shutdown` 
+```
+sonic(conf-if-po1)# shutdown
+```
+
+#### Configures an IPv4 address 
+`ip address <ip-address with mask> | no ip address <ip-address>`
+```
+sonic(conf-if-po1)# ip address 2.2.2.2/24
+```
+#### Remove IPv4 address 
+`no ip address <ip-address>`
+```
+sonic(conf-if-po1)# no ip address 2.2.2.2
+```
```

Comment:
Please add a new command to configure the 'fallback' feature.
Current fallback configuration is shown below:
===============
root@sonic:/home/admin# config portchannel add PortChannel1
root@sonic:/home/admin# config portchannel add PortChannel2 --fallback=true
root@sonic:/home/admin# show runningconfiguration all
<<removing non-related config >>
{
    "PORTCHANNEL": {
        "PortChannel1": {
            "admin_status": "up",
            "mtu": "9100"
        },
        "PortChannel2": {
            "admin_status": "up",
            "fallback": "true",
            "mtu": "9100"
        }
    },
<<-->>
===============